### ISSUE TITLE
ERROR: Cannot find module '@schematics/angular/utility/workspace'

### ISSUE THREAD

**Issue by AnassAzeroual** [maciejtreder/ng-toolkit#731]
## issue in  installation package ng-toolkit
<pre><code>

Installing packages for tooling via npm.
[bugsnag] Loaded!
ERROR: Cannot find module '@schematics/angular/utility/workspace'
ERROR: If you think that this error shouldn't occur, please fill up bug report here: https://github.com/maciejtreder/ng-toolkit/issues/new
INFO: stacktrace has been sent to tracking system.
Nothing to be done.

</code></pre>

**Comment by KingDarBoja** [comment]
Hello @AnassAzeroual,

Can you provide more info like as sample repository or share your settings like `package.json` and the reproduction steps?

Cheers!

**Comment by AnassAzeroual** [comment]
Hi Bojato,


sorry, i can not provide project repository, but i can give you
package.json, i hope it will come in handy

Le ven. 22 nov. 2019 à 14:43, Manuel Bojato <notifications@github.com> a
écrit :

**Comment by KingDarBoja** [comment]
@AnassAzeroual Yes, a `package.json` and also reproduction of the problem could be helpful.

**Comment by poojit123** [comment]
https://github.com/maciejtreder/ng-toolkit/issues/731#issue-527168312

I am facing same problem. Please any body find out this issue.

**Comment by KingDarBoja** [comment]
Could be helpful to me if you can mention the ng-toolkit version being listed on your package.json.

**Comment by itechdriven** [comment]
I am also facing same problem. Please any body find out this issue.

**Comment by JosFran21** [comment]
Hello guys.
I have the same problem, have you managed to solve it?

**Comment by KingDarBoja** [comment]
This is getting fixed at #826. This is caused by how Angular schematics API changes between versions.
